packages/next/plugins/with-nx.ts

```diff
@@ -156,6 +156,15 @@ function withNx(
   _nextConfig = {} as WithNxOptions,
   context: WithNxContext = getWithNxContext()
 ): NextConfigFn {
+  // If this is not set user will see compile errors in Next.js 13.4.
+  // See: https://github.com/nrwl/nx/issues/16692, https://github.com/vercel/next.js/issues/49169
+  // TODO(jack): Remove this once Nx is refactored to invoke CLI directly.
+  forNextVersion('>=13.4.0', () => {
+    process.env['__NEXT_PRIVATE_PREBUNDLED_REACT'] =
+      // Not in Next 13.3 or earlier, so need to access config via string
+      _nextConfig.experimental['serverActions'] ? 'experimental' : 'next';
```

Comment:
@jaysoo If `serverActions` is not specified on the experimental object, then it will throw an error because of undefined.

Reply:
Yes, I had the same error as @ruslanguns. It should be first checked if experimental object exists in the config, something like `_nextConfig.experimental && _nextConfig.experimental['serverActions']`

Reply:
@dpnolte just dropped a PR with your suggested fix https://github.com/nrwl/nx/pull/16846